meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx8/boot.cmd

```diff
@@ -5,28 +5,46 @@ echo "Using freescale_${fdt_file}"
 # Default boot type and device
 setenv bootlimit 3
 setenv devtype mmc
-setenv devnum ${mmcdev}
+setenv devnum 0
+setenv bootpart 1
+setenv rootpart 2
 setenv hdmi_image hdmitxfw.bin
 setenv m4_0_image core0_m4_image.bin
 setenv m4_1_image core1_m4_image.bin
+setenv ramdisk_addr_r 0x8a000000
```

Comment:
any reason why we redefine it here?
in `include/configs/apalis-imx8.h ` it's there already:
```
 84 #define MEM_LAYOUT_ENV_SETTINGS \                                               
 85         "fdt_addr_r=0x83000000\0" \                                             
 86         "hdp_addr=0x9c000000\0" \                                               
 87         "kernel_addr_r=0x80280000\0" \                                          
 88         "ramdisk_addr_r=0x8a000000\0" \ 
```

Reply:
This was added when comparing the boot scripts between apalis-imx8 and imx8qm-mek.
I'll drop as it makes for easier comparison but not needed.

Reply:
Dropped this change